linux_admin.gemspec

```diff
@@ -38,7 +37,7 @@ registration, updates, etc.
   spec.add_dependency "awesome_spawn",        "~> 1.3"
   spec.add_dependency "inifile"
   spec.add_dependency "more_core_extensions", "~> 3.0"
-  spec.add_dependency "nokogiri",             ">= 1.8.2", "~> 1.8"
+  spec.add_dependency "nokogiri",             ">= 1.8.5", "!=1.10.0", "!=1.10.1", "!=1.10.2", "<2"
```

Comment:
@jrafanie @Fryguy this seems to solve the problem, but is there a more concise way of doing this? Alternatively, is there some documentation of the available operators? The only thing I can find is https://guides.rubygems.org/patterns/#pessimistic-version-constraint and that doesn't mention the `!=` at all.

Reply:
Yay, I don't know.  It's smoke and mirrors... and some magic.